sdk/notificationhubs/notification-hubs/package.json

```diff
@@ -23,7 +23,7 @@
     "build": "npm run build:test && dev-tool run extract-api",
     "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"samples-dev/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf --glob dist dist-* temp types *.tgz *.log",
-    "execute:samples": "dev-tool samples run samples-dev",
+    "execute:samples": "tsx",
```

Comment:
what does tsx alone do here?

Reply:
It will run any sample as is for ESM instead of using dev-tool to do it.

Reply:
just it alone!? like you don't need to pass file paths?